content/en/docs/components/pipelines/v2/quickstart.md

```diff
@@ -1,16 +1,34 @@
 +++
 title = "Quickstart"
-description = "Getting started with Kubeflow Pipelines"
+description = "Get started with Kubeflow Pipelines"
 weight = 3
                     
 +++
+
+{{% stable-status %}}
+
 <!-- TODO: add UI screenshots for final pipeline -->
-In this tutorial we will deploy a KFP standalone deployment into an existing Kubernetes cluster, create and run a simple pipeline using the KFP SDK, view the pipeline on the KFP Dashboard, then create a more involved machine learning pipeline that uses additional KFP features.
+This tutorial helps you get started with KFP by doing the following:
+
+1. [Deploy a KFP standalone instance into an existing Kubernetes cluster.](#step-1-deploy-a-kfp-standalone-instance-into-an-existing-kubernetes-cluster)
+2. [Create and run a simple pipeline using the KFP SDK.](#step-2-create-and-run-a-simple-pipeline-using-the-kfp-sdk)
+3. [View the pipeline on the KFP Dashboard.](#step-3-view-the-pipeline-in-the-kfp-dashboard)
+4. [Create a more involved machine learning (ML) pipeline that uses additional KFP features.](#step-4-build-a-more-advanced-ml-pipeline-that-uses-additional-kfp-features)
+
+For detailed guidance about each step, see the following sections.
+
```

Comment:
Suggested reorganizing this topic like the following quick start guide:

https://developers.google.com/docs/api/quickstart/python